ui/src/app/kvm/components/KVMConfigurationCard/KVMConfigurationCard.tsx

```diff
@@ -42,6 +47,7 @@ type Props = {
 };
 
 const KVMConfigurationCard = ({ pod, zoneDisabled }: Props): JSX.Element => {
+  const [hasUpdated, setHasUpdated] = useState(false);
```

Comment:
**QA note**
This results in different behaviour to the same interactions across time.

Before first submission saving an unchanged (equal to what's saved on the back-end) value is disallowed, but allowed on every subsequent change.

We should probably be updating the initial value on after submit to keep the same behaviour across interactions. I'm okay to leave this as is If it happens to be too complex to fix, but would be good to change.

Reply:
Yeah I ended up taking the "easy" route to avoid the complexity of having to track when the underlying `pod` object updates. Figured the behaviour was inconsistent but at least it wasn't buggy. But I actually forgot that `Formik` provides an `enableReinitialize` prop that solves this problem entirely, so that whenever `initialValues` changes the form is reset. Whoops!